control-plane/pkg/core/config/utils.go

```diff
@@ -66,11 +66,11 @@ func EgressConfigFromDelivery(
 		if destination.Ref != nil && destination.Ref.Namespace == "" {
 			destination.Ref.Namespace = parent.GetNamespace()
 		}
-		deadLetterSinkURL, err := resolver.URIFromDestinationV1(ctx, *delivery.DeadLetterSink, parent)
+		deadLetterSinkURL, err := resolver.AddressableFromDestinationV1(ctx, *delivery.DeadLetterSink, parent)
```

Comment:
```suggestion
		deadLetterSinkAddr, err := resolver.AddressableFromDestinationV1(ctx, *delivery.DeadLetterSink, parent)
```

NIT, update variable names, as it is a `addressable` now and not a `URL`